baseplate/live_data/writer.py

```diff
@@ -0,0 +1,119 @@
+"""Write a file's contents to a node in ZooKeeper."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import difflib
+import logging
+import sys
+
+from kazoo.exceptions import NoNodeError, BadVersionError
+
+from .._compat import configparser
+from ..secrets import secrets_store_from_config
+from .zookeeper import zookeeper_client_from_config
+
+
+logger = logging.getLogger(__name__)
+
+
+class WriterError(Exception):
+    exit_status = 0
```

Comment:
Is it because NoChangesError is a zero? I think I'd rather have the inverted logic. Errors should default to "bad" things unless we explicitly say the error doesn't matter.